src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -215,11 +213,21 @@ std::string postQueryContext
     return answer;
   }
 
-  s = xmlTreat(cleanPayload, ciP, &parseData, RtQueryContextResponse, "queryContextResponse", NULL, &errorMsg);
+  //
+  // NOTE
```

Comment:
Good point. I didn't realice that the comment was in the "common" postQueryContext logic. I though that I was in a per-convop specific part.

I don't like too much manipulating ciP as I see it as an object representing the "connection environment", but I think it is the most pragmatic way of doing. However, maybe I'm not objective... you know I don't like too much ciP ;)

NTC